osu.Game/Localisation/MouseSettingsStrings.cs

```diff
@@ -54,6 +54,11 @@ public static class MouseSettingsStrings
         /// </summary>
         public static LocalisableString CursorSensitivity => new TranslatableString(getKey(@"cursor_sensitivity"), @"Cursor sensitivity");
 
+        /// <summary>
+        /// "This setting currently has issues on your platform. It is recommended to adjust sensitivity externally and keep this disabled for now."
```

Comment:
```suggestion
        /// "This setting has known issues on your platform. If you encounter problems, it is recommended to adjust sensitivity externally and keep this disabled for now."
```
https://github.com/ppy/osu-localisation-analyser/issues/37